src/rviz/visualization_manager.cpp

```diff
@@ -178,6 +178,10 @@ VisualizationManager::VisualizationManager( RenderPanel* render_panel, WindowMan
                                    "RViz will try to render this many frames per second.",
                                    global_options_, SLOT( updateFps() ), this );
 
+  headlight_property_ = new BoolProperty( "Headlight", true,
```

Comment:
I would suggest to name this "Default Light" (headlight makes less sense when considering a large-scale visualisation, and suggests it's a spotlight). Unless @wjwwood you have another suggestion for what to call the global directional light?